scripts/validate_string_concatenation.py

```diff
@@ -111,6 +111,30 @@ def strings_with_wrong_placed_space(
     msg : str
         Explenation of the error.
     """
+
+    def idk_how_to_call_this_function(first_line: str, second_line: str) -> bool:
```

Comment:
I really don't know how to call this :smile:

Reply:
Naming things and cache invalidation are the two most challenging problems in CS. :)

Sorry, the expression is too difficult to understand. Do you mind using something like:
```python
if first_line.endswith(r'\n'):
   return False
if second_line.ends_with('  '):
    return False
...
```
Happy to help find a name when I understand what's this doing. Also, may be worth adding an example sections, since this function seems somehow complex.

You have couple of typos in the docsting, and I think `r'\n'` is more readable than `'\\n'`.

Thanks!